onelinerizer/__init__.py

```diff
@@ -0,0 +1 @@
+from onelinerizer import onelinerize
```

Comment:
This could be an explicitly relative import (`from .onelinerizer import onelinerize`).